src/components/PupilComponents/QuizOrderAnswer/QuizOrderAnswer.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  OakBox,
+  OakDraggableFeedback,
+  OakDroppable,
+  OakQuizOrder,
+  OakQuizOrderProps,
+} from "@oaknational/oak-components";
+import { useMemo, useState } from "react";
+import { isArray } from "lodash";
+
+import { isOrderAnswer } from "../QuizUtils/answerTypeDiscriminators";
+
+import { useQuizEngineContext } from "@/components/PupilComponents/QuizEngineProvider";
+import { useInitialChange } from "@/components/PupilComponents/QuizUtils/useInitialChange";
+import OakError from "@/errors/OakError";
+
+export type QuizOrderAnswerProps = {
+  onInitialChange?: () => void;
+  onChange?: () => void;
+};
+
+export const QuizOrderAnswer = ({
+  onChange,
+  onInitialChange,
+}: QuizOrderAnswerProps) => {
+  const { handleOnChange: handleInitialChange } = useInitialChange({
+    onChange,
+    onInitialChange,
+  });
+  const { currentQuestionData, questionState, currentQuestionIndex } =
+    useQuizEngineContext();
+  invariant(currentQuestionData, "currentQuestionData is not defined");
+  const answers = currentQuestionData.answers;
+  const questionUid = currentQuestionData.questionUid;
+  const feedback = questionState[currentQuestionIndex]?.feedback;
+
+  invariant(
+    answers && isOrderAnswer(answers),
+    `current '${questionUid}' is not an order question`,
+  );
+
+  /**
+   * Memoise the randomised order of items to preserve it across renders
+   */
+  const initialItems = useMemo(() => {
+    const originalItems = answers.order.map((item, index) => {
+      const label = item?.answer?.[0]?.text;
+      invariant(
+        label,
+        `label is missing for option in question '${questionUid}'`,
+      );
+
+      return {
+        id: (index + 1).toString(),
+        label,
+      };
+    });
+    let currentItems = originalItems.slice();
+
+    // Randomise items while ensuring that the random order is not the same as the original order
+    while (
+      originalItems.length > 1 &&
+      JSON.stringify(originalItems) === JSON.stringify(currentItems)
+    ) {
+      currentItems = currentItems.sort(() => 0.5 - Math.random());
+    }
```

Comment:
The requirements are for the answers to be randomised — however there is a chance that the random order will be the same as the original order (especially when there are only a handful of items). This ensures that the randomisation will run until there is at least 1 item out of order.

Reply:
_Placeholder for witty pseudo philosophical  quip on randomness._
But yeah it would be weird if there was no ordering to be done.